src/Libs/Secret-1/Secret-1.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>GirCore.Secret-1</PackageId>
+    <RootNamespace>Secret</RootNamespace>
+    <Description>C# bindings for Secret.</Description>
```

Comment:
Perhaps we should call this "C# bindings for libsecret." I think this name is better known.